### PR TITLE
Fix packaging

### PR DESCRIPTION
**Pull request by EvaSDK**
As imaplib2 started to be used in offlineimap2, I had to apply the attached changes to make it installable.

### CODE REVIEW

File: setup.cfg [hunk 1]
```diff
@@ -1,2 +1,25 @@
+[metadata]
+name = imaplib2
+version = attr: imaplib2.imaplib2.__version__
+description = A threaded Python IMAP4 client.
+long_description = file: README
+long_description_content_type = text/plain
+url = https://github.com/bcoe/imaplib2
```
Comment (chris001) [inline setup.cfg:7]: Update:  `url = https://github.com/imaplib2/imaplib2`

Comment (chris001) [reply setup.cfg:7]: Or `url = https://github.com/jazzband/imaplib2`

File: README [hunk 2]
```diff
@@ -1,11 +1,14 @@
...
+Threaded Python IMAP4 client version 3.05.
 
 Based on RFC 3501 and original imaplib module.
 
 This is a version of imaplib that uses threads to allow full use of the
 IMAP4 concurrency features, and to de-couple a user of imaplib from i/o
 lags, except where explicitly allowed.
 
-Documented in imaplib2.html
+This is a BETA release of a version converted for native Python3 use.
```
Comment (graingert) [inline README]:
```suggestion
Documented in imaplib2.html
```

File: README [hunk 3]
```diff
@@ -1,11 +1,14 @@
...
 
-Install imaplib2.py into your Python library path.
+Documented in imaplib.html.py3, but note the removal of argument quoting
+compared with the py2 version of imaplib2.
+
+Install imaplib2.py3 into your Python library path (rename as imaplib2.py).
```
Comment (graingert) [inline README]:
```suggestion
```

File: setup.cfg (hunk 1 above)
Comment (graingert) [reply setup.cfg:7]:
```suggestion
url = https://github.com/jazzband/imaplib2
```

**Pull request merged**